framework/src/auxkernels/FunctorElementalGradientAux.C

```diff
@@ -25,6 +25,18 @@ FunctorElementalGradientAuxTempl<is_ad>::validParams()
   params.addParam<MooseFunctorName>("factor", 1, "A factor to apply on the functor");
   params.addParam<MaterialPropertyName>(
       "factor_matprop", 1, "A (regular) material property factor to apply on the functor");
+
+  // We need some ghosting for the Finite Volume Fields (we use neighbor information to compute
+  // gradient)
+  params.addParam<unsigned short>("ghost_layers", 1, "The number of layers of elements to ghost.");
+  params.addRelationshipManager(
+      "ElementSideNeighborLayersAux",
```

Comment:
```suggestion
      "ElementSideNeighborLayers",
```

Reply:
I probably hit every possible mistake on defining this RM haha, will correct this thanks Alex!

Reply:
I'm really confused why we can compile with this line the way it is

Reply:
Okay, I'll look into the RMs to make sure I can add them properly.